sql/core/src/main/scala/org/apache/spark/sql/DataFrame.scala

```diff
@@ -1632,7 +1632,7 @@ class DataFrame private[sql](
    * be the target of an `insertInto`.
    *
    * Also note that while this function can persist the table metadata into Hive's metastore,
-   * the table will NOT be accessible from Hive, until SPARK-7550 is resolved.
+   * if spark.sql.hive.writeDataSourceSchema=true in configuration.
```

Comment:
After the change, these comments don't make grammatical sense, and I'm not sure what information they are trying to convey.